scripts/run_group.py

```diff
@@ -0,0 +1,115 @@
+import json
+import os
+import subprocess
+import sys
+
+DJANGO_HOME = os.path.join(os.getcwd(), "testrepo")
+os.environ["DJANGO_HOME"] = DJANGO_HOME
+os.environ["PYTHONPATH"] = (
+    f"{DJANGO_HOME}:{DJANGO_HOME}/tests:{DJANGO_HOME}/tests/singlestore_settings:"
+    + os.environ.get("PYTHONPATH", "")
+)
+
+# Default Django apps
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_ADMIN"] = "ROWSTORE REFERENCE"
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_AUTH"] = "ROWSTORE REFERENCE"
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_CONTENTTYPES"] = "ROWSTORE REFERENCE"
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_SITES"] = "ROWSTORE REFERENCE"
+
+# 12 many-to-many fields
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_PREFETCH_RELATED"] = "ROWSTORE REFERENCE"
+
+# many-to-many to self
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_M2M_RECURSIVE"] = "ROWSTORE REFERENCE"
+
+# lots of many-to-many fields
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_ADMIN_VIEWS"] = "ROWSTORE REFERENCE"
+
+# unique keys and many-to-many
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_FOREIGN_OBJECT"] = "ROWSTORE REFERENCE"
+
+# unique keys
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_AUTH_TESTS"] = "ROWSTORE REFERENCE"
+
+# abstract models
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_MANY_TO_MANY"] = "ROWSTORE REFERENCE"
+
+# models with unique keys and m2m fields
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_FIXTURES_REGRESS"] = "ROWSTORE REFERENCE"
+
+# m2m-related issue
+os.environ["DJANGO_SINGLESTORE_TABLE_STORAGE_TYPE_DELETE"] = "ROWSTORE REFERENCE"
+
+# OneToOne relationships, disable enforced unique
+os.environ["DJANGO_SINGLESTORE_NOT_ENFORCED_UNIQUE_QUERIES"] = "1"
+
+
+def setup_module(module, ):
+    """Runs a setup SQL script for the given module if it exists."""
+    sql_file = f"scripts/setup_section/{module}.sql"
+    if not os.path.exists(sql_file):
+        # No setup script for this module, which is fine.
+        return
+
+    print(f"--- Setting up for module '{module}' ---")
+    password = os.environ.get("SINGLESTORE_PASSWORD", "")
+    mysql_cmd = [
```

Comment:
I'd better use singlestore python connector to execute queries so that we don't rely on mysql cli being available and don't spawn a new process